finalPrep/annotations.js

```diff
@@ -2,41 +2,60 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+// Groups the properties of the Build a Bear.
 function buildABear(name, age, fur, clothes, specialPower) {
+  // Greeting string using bear's name.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  // Gives name and age of bear.
   var demographics = [name, age];
+  // Logs string about bear's special power.
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  // Object designating parts of building a bear.
   var builtBear = {
+    // Info refering to the demographics given.
     basicInfo: demographics,
+    // The clothes the bear will have.
     clothes: clothes,
+    // Color of the bear's fur.
     exterior: fur,
+    // Cost of the bear.
     cost: 49.99,
+    // Different voicelines the bear will have.
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+    // Is the bear cuddly?
```

Comment:
As you continue to practice annotations at Turing, consider how you can introduce some technical vocabulary to your annotations.  For example, lines 16-24 are all the key-value pairs (or properties) on the `builtBear` object.